src/api/annotation/AnnotationAPI.js

```diff
@@ -172,17 +172,19 @@ export default class AnnotationAPI extends EventEmitter {
                 name: contentText,
                 domainObject: targetDomainObject,
                 annotationType,
-                tags: [],
+                tags: [tag],
```

Comment:
either create the annotation with the new tag, or mutate, but don't try to both at once.